src/main/haskell/kore/src/Kore/Builtin/Set.hs

```diff
@@ -355,3 +374,107 @@ lookupSymbolDifference
     :: KoreIndexedModule attrs
     -> Either (Kore.Error e) (Kore.SymbolOrAlias Object)
 lookupSymbolDifference = Builtin.lookupSymbol "SET.difference"
+
+{- | Simplify the conjunction of two concrete Set domain values.
+
+    The sets are assumed to have the same sort, but this is not checked. If
+    multiple sorts are hooked to the same builtin domain, the verifier should
+    reject the definition.
+ -}
+
+unify
+    :: forall level variable m p expanded proof.
+        ( OrdMetaOrObject variable, ShowMetaOrObject variable
+        , Kore.SortedVariable variable
+        , MonadCounter m
+        , MetaOrObject level
+        , Hashable variable
+        , FreshVariable variable
+        , p ~ Kore.PureMLPattern level variable
+        , expanded ~ ExpandedPattern level variable
+        , proof ~ SimplificationProof level
+        )
+    => MetadataTools level StepperAttributes
+    -> (p -> p -> Result (m (expanded, proof)))
+    -> (p -> p -> Result (m (expanded, proof)))
+unify
+    tools@(MetadataTools.MetadataTools { symbolOrAliasSorts })
+    _ -- not used now. Should remove? Or may be useful later?
```

Comment:
This argument may be useful if we adopt a [semi-concrete](https://trello.com/c/pYiWKgdg) representation for sets. Otherwise, it is only used to keep the signature of this function the same as the signature of the other unification cases; but that is important enough to keep it!